project_event/tests/test_project_event_task.py

```diff
@@ -458,3 +458,48 @@ def test_190_main_task_activity_calendar_event_clone_state(self):
         for child in self.activity_1.child_ids:
             calendar_event = child.get_calendar_event()
             self.assertEqual(calendar_event.state, 'draft')
+
+    def test_200_compute_actual_total_time(self):
+        activity_vals_1 = {
+            'name': 'Sample activity 1',
```

Comment:
REal date start and real date end should be filled for a task not for an activity

Reply:
@rimbendhaou Changed and done